cheroot/test/test_server.py

```diff
@@ -439,3 +441,52 @@ def many_open_sockets(request, resource_limit):
         # Close our open resources
         for test_socket in test_sockets:
             test_socket.close()
+
+
+@pytest.mark.parametrize(
+    ('minthreads', 'maxthreads', 'error'),
+    (
+        (-1, -1, ValueError),
+        (-1, 0, ValueError),
+        (-1, 1, ValueError),
+        (-1, 2, ValueError),
+        (0, -1, ValueError),
+        (0, 0, ValueError),
+        (0, 1, ValueError),
+        (0, 2, ValueError),
+        (1, -1, None),
+        (1, 0, ValueError),
+        (1, 1, None),
+        (1, 2, None),
+        (2, -1, None),
+        (2, 0, ValueError),
+        (2, 1, ValueError),
+        (2, 2, None),
+    ),
+)
+def test_threadpool_threadnum_validation(minthreads, maxthreads, error):
+    kwargs = dict(
```

Comment:
I think this extra var adds mental overhead because it is detached from a single tested initializer invocation.

Reply:
see other comment